usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3674,6 +3674,17 @@ CK_RV token_specific_aes_xts_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
         goto retry;
     }
 
+    /*
+     * Compare the encrypted key material to ensure that the 2 key parts are
+     * not the same.
+     * A CCA AES-DATA key blob contains the encrypted key material at
+     * offset 16, with a length of 32 bytes.
```

Comment:
Is this always the case? Are AES-256 XTS keys not supported by CCA?

Reply:
Remember that CCA does not support XTS at all. Tats why we build an XTS key out of 2 single AES keys.

A single CCA AES key token always have an 32 bytes encrypted key, regardless of the key-size of the underlying AES key. Simply because the AES-master key is an AES-256 bit key, so the resulting encrypted key is always 32 bytes.